roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -56,3 +56,56 @@
     - name: Set variable
       ansible.builtin.set_fact:
         cifmw_test_operator_tempest_exclude_list: "{{ list_skipped.skipped_tests | join('\n') }}"
+
+- name: Ensure a secret for the cifmw private key file exists
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    state: present
+    wait: true
+    definition:
+      apiVersion: v1
+      kind: Secret
+      type: Opaque
+      metadata:
+        name: "{{ cifmw_test_operator_controller_priv_key_secret_name }}"
+        namespace: "{{ cifmw_test_operator_namespace }}"
+      data:
+        ssh-privatekey: "{{ lookup('file', '~/.ssh/id_cifw') | b64encode }}"
+  when:
+    - not cifmw_test_operator_dry_run | bool
+    - cifmw_test_operator_tempest_ssh_key_secret_name is not defined
+
+- name: Add SSHKeySecretName section to tobiko CR
+  ansible.builtin.set_fact:
+    cifmw_test_operator_tempest_config: >-
```

Comment:
Keep in mind that is someone passes the `cifmw_test_operator_tempest_config` as variable in the ansible-playbook call, using `-e @<path-that-contains-that>` that value won't be updated and this set_fact and its combine wont' work, as the following tasks will still see the "old" value comming from the file passed in the cli.

Reply:
Thank you for pointing this out. Hopefully addressed in the second patch in this PR. If you think that there is nicer solution than let me know please:).